lib/startup/loader_progress.js

```diff
@@ -34,43 +42,47 @@ function LoaderProgress(config) {
 
 }
 
-LoaderProgress.prototype.listen = function (callback) {
-
-  this.__progressServer = http.createServer(function (req, res) {
+function _catchStreamError(err, res) {
+  res.writeHead(404, {"Content-Type": "text/plain"});
+  res.write("404 Not Found: " + err + "\n");
+  res.end();
+  return;
+}
 
-    if (req.url == '/log') {
-      res.write(JSON.stringify(__messageLog));
-      return res.end();
-    }
+function _matchPathRoute(url) {
+  if (!url.match(LOADER_PATH.match)) return false;
+  var targetFilename = url.replace(LOADER_PATH.match, LOADER_PATH.path);
+  if (os.platform() === 'win32') {
```

Comment:
I've just checked that on windows, path.normalize will replace every '/' with '\\'